build/transpile.js

```diff
@@ -1355,7 +1357,7 @@ class Transpiler {
 
         const tsBodyIntellisense = tsBaseError + '\n\n    ' + tsErrors.join ('\n    ') + '\n\n'
 
-        const tsFilename = './ccxt.d.ts'
+        const tsFilename = isPro ? './ccxt.pro.d.ts' : './ccxt.d.ts'
```

Comment:
Can we do this in CCXT Pro instead of CCXT ?

Reply:
should we copy the whole method into pro?

Reply:
isn't this? https://github.com/kroitor/ccxt.pro/blob/master/ccxtpro.php